WooCommerce/Classes/Authentication/AuthenticationManager.swift

```diff
@@ -31,6 +31,7 @@ class AuthenticationManager: Authentication {
                                                                 wpcomScheme: ApiCredentials.dotcomAuthScheme,
                                                                 wpcomTermsOfServiceURL: WooConstants.URLs.termsOfService.rawValue,
                                                                 wpcomAPIBaseURL: Settings.wordpressApiBaseURL,
+                                                                whatIsWPComURL: WooConstants.URLs.whatIsWPComURL.rawValue,
```

Comment:
do we want to run an AB test on this change as an experiment?

Reply:
We have decided to release this without A/B testing. Discussed here - p1657185371754549/1657093471.680409-slack-CGPNUU63E